Laura/pvc update (#986)
elevenlabs/elevenlabs-docs: fern/docs/pages/product-guides/voices/voice-cloning/professional-voice-cloning.mdx

    </Frame>

    ### Process your audio

    Once your audio samples have been uploaded, you can process them to improve the quality. You can remove any background noise, and you can also separate out different speakers, if your audio includes more than one speaker. To access these options, click the **Audio settings** button next to the clip you want to process.

    <Frame background="subtle">
      ![Create a new Professional Voice Clone](/assets/images/product-guides/voices/voice-cloning/voice-pvc-creation-settings.webp)
    </Frame>

    ### Verify your voice

    Once everything is recorded and uploaded, you will be asked to verify your voice. To ensure a smooth experience, please try to verify your voice using the same or similar equipment used to record the samples and in a tone and delivery that is similar to those present in the samples. If you do not have access to the same equipment, try verifying the best you can. If it fails, you can either wait 24 hours to try verification again, or reach out to support for help.

    ### Wait for your voice to complete fine tuning

>     Before you can use your voice, it needs to complete the fine tuning process. You can check the status of your voice in My Voices while it's processing. You'll also be notified when it's ready to use.

Reviewer: ```suggestion
    Before you can use your voice, it needs to complete the fine tuning process. You can check the status of your voice in My Voices while it's processing. You'll be notified when it's ready to use.
```

### Provide enough material

> Make sure you have enough material to clone the voice properly. The bare minimum we recommend is 30 minutes of audio, but for the optimal result and the most accurate clone, we recommend closer to 2-3 hours of audio. You might be able to get away with less, but at that point, we can’t vouch for the quality of the resulting clone.

Reviewer: ```suggestion
Make sure you have enough material to clone the voice properly. The bare minimum we recommend is 30 minutes of audio, but for the optimal result and the most accurate clone, we recommend closer to 2-3 hours of audio. The more audio provided the better the quality of the resulting clone.
```
